.github/workflows/e2e.yml

```diff
@@ -0,0 +1,33 @@
+name: E2E - Test
+
+on: push
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      preview_url: ${{ steps.waitForVercelPreviewDeployment.outputs.url }}
+    steps:
+      - name: Wait for Vercel preview deployment to be ready
+        uses: patrickedqvist/wait-for-vercel-preview@v1.2.0
+        id: waitForVercelPreviewDeployment
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          max_timeout: 600
+  test:
+    needs: setup
+    name: Run Playwright tests
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Prepare Playwright
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+      - run: yarn
+      - run: npx playwright install --with-deps
+      - name: Run tests
+        run: yarn test:e2e
+        env:
+          BASE_URL: ${{ needs.setup.outputs.preview_url }}
```

Comment:
Right now the example site is building against `next-themes@beta` instead of the code inside of `src/`, in a future PR we should make it build the package and use that.

Reply:
I think we could do this with next-transpile-modules 🤔